certified-connectors/Oneflow/apiProperties.json

```diff
@@ -18,8 +18,8 @@
         "type": "string",
         "uiDefinition": {
           "displayName": "Email address",
-          "description": "Oneflow user email address",
-          "tooltip": "Provide Oneflow user email address",
+          "description": "Login of Oneflow user the steps will be executed under",
```

Comment:
@DmitryKleshchov please make sure everything is grammatically correct and is free of typographical errors. You can refer to the [guidelines for creating strings](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).